include/eosio/vm/execution_context.hpp

```diff
@@ -268,6 +268,11 @@ namespace eosio { namespace vm {
          get_operand_stack().eat(0);
       }
 
+// Calling execute() with no `args` (i.e. `execute(host_type,jit_visitor,uint32_t)`) results in a "statement has no effect
+// [-Werror=unused-value]" warning. Dissable for this function.
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wunused-value"
+
       template <typename... Args>
       inline std::optional<operand_stack_elem> execute(host_type* host, jit_visitor, uint32_t func_index, Args... args) {
```

Comment:
Can we use: `[[maybe_unused]]` instead?

Reply:
@heifner The attribute `maybe_unused` did not work when tried. Maybe I misplaced it? 

Arguably, we could could create the specialized template that takes no args, but that would give us another function to maintain.

Reply:
Should go before param `Args... args`

Reply:
The scope of ignore was restricted to a single line.